web/README.md

```diff
@@ -5,6 +5,11 @@ In order to launch SHARK-web, from the root SHARK directory, run:
 IMPORTER=1 ./setup_venv.sh
 source shark.venv/bin/activate
 cd web
+```
+Download [RRDB_ESRGAN_x4.pth](https://drive.google.com/file/d/1TPrz5QKd8DHHt1k8SRtm6tMiPjz_Qene/view)
+
+Then run:
```

Comment:
Can you make it a command by using utilities like `wget`?

Reply:
I've made the appropriate change using `gdown` instead since the file is hosted at a Google Drive.
Let me know if that sounds good to you.